libs/partners/nvidia-trt/tests/integration_tests/test_llms.py

```diff
@@ -17,7 +17,7 @@ def test_stream() -> None:
 
 @pytest.mark.skip(reason="Need a working Triton server")
 async def test_astream() -> None:
-    """Test streaming tokens from OpenAI."""
+    """Test streaming tokens from nvidia-trt."""
```

Comment:
likewise here,

and they go all uppercase
```suggestion
    """Test streaming tokens from NVIDIA TRT."""
```